frontend/public/module/k8s/installed-operators.ts

```diff
@@ -0,0 +1,6 @@
+import * as _ from 'lodash-es';
+
+import { K8sResourceKind } from '../../module/k8s';
+
+export const installedOperatorDisplayName = (installedOperator: K8sResourceKind): string =>
```

Comment:
These are ClusterServiceVersion resources, so we should use that terminology in all names/filenames.